src/router.ts

```diff
@@ -2,9 +2,28 @@ import { identity } from 'lodash'
 import * as Router from 'koa-router'
 import * as send from 'koa-send'
 import { readdirSync } from 'fs'
-import { parse } from 'path'
+import { parse, ParsedPath } from 'path'
 import * as handlers from './handlers'
 
+declare module 'koa-router' {
+  export interface IRouterContext {
+    render(view: string, data?: any): any
+  }
+}
+
+// Sychronously yield all files in the given directory, searching recursively
+function * files(dir: string): IterableIterator<ParsedPath & { full: string }> {
+  for (const name of readdirSync(process.cwd() + dir)) {
```

Comment:
Using `process.cwd()` and the `+` sign is risky. Let's use `__dirname` and `path.join`